src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/DeleteIT.java

```diff
@@ -77,12 +68,76 @@ public void deleteWithConditionTest() throws IOException, InterruptedException {
         // To prevent flakiness, the minimum value of 2000 msec works fine.
         Thread.sleep(2000);
 
-        selectQuery = String.format(
-            Locale.ROOT,
-            "SELECT * FROM %s/phrase",
+        selectQuery = StringUtils.format("SELECT * FROM %s/phrase", TestsConstants.TEST_INDEX_PHRASE);
+
+        response = executeRequest(makeRequest(selectQuery));
+        assertThat(getTotalHits(response), equalTo(5));
+    }
+
+    @Test
+    public void deleteAllWithJdbcFormat() throws IOException, InterruptedException {
+        String selectQuery = StringUtils.format("SELECT * FROM %s/account", TestsConstants.TEST_INDEX_ACCOUNT);
+        JSONObject response = executeRequest(makeRequest(selectQuery));
+        int totalHits = getTotalHits(response);
+
+        String deleteQuery = StringUtils.format("DELETE FROM %s/account", TestsConstants.TEST_INDEX_ACCOUNT);
+
+        response = new JSONObject(executeQuery(deleteQuery, "jdbc"));
+        System.out.println(response);
+        assertThat(response.query("/schema/0/name"), equalTo("deleted_rows"));
+        assertThat(response.query("/schema/0/type"), equalTo("long"));
+        assertThat(response.query("/datarows/0/0"), equalTo(totalHits));
+        assertThat(response.query("/total"), equalTo(1));
+        assertThat(response.query("/status"), equalTo(200));
+        assertThat(response.query("/size"), equalTo(1));
```

Comment:
Should assertion for `total` and `size` be `equalTo(totalHits)`? And any reason not to use `total` or `size` as deleted rows directly?

Reply:
Yes, we could do it. I made my changes based on the solution proposed #131 .  Based on your recommendation should the response look like the following:

```
{
  "schema": [],
  "total": <number of deleted rows> ,
  "datarows": [],
  "size": <number of deleted rows> >,
  "status": 200
}
```

Should we get rid of `schema` and `datarows` altogether?

Reply:
Yeah, maybe. Or just leave them empty if any issue.

Reply:
I realized , we need the `schema` with` deleted_rows` column, because when converting to `csv` we don't emit the `total` and `size` values. Since we plan to derive the `csv` output directly from the common representation (currently used for `jdbc` format), in absence of `deleted_rows` column , the `csv` output would be empty.

Reply:
Sure, let's follow your current format. Thanks!